project/src/main/java/com/google/sps/data/FakePlaceDetailsRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
```

Comment:
This can also reside as a nested class of the fake generator, instead of a class of its own.

Reply:
I didn't think about, it does make much more sense to put it in the test dierctory :)
I added the fake requests as public static nested classes - static because they are not related to an instance of the generator, and public because they are accessed from the tester. Let me know if this is fine.

Reply:
👍